train/train_process.py

```diff
@@ -106,24 +104,24 @@ def _self_play(self, model_path, out_dir):
             "--out-dir", out_dir,
             "--sim-count", str(self.cfg["mcts"]["sim_count"]),
             "--explore-factor", str(self.cfg["mcts"]["explore_factor"]),
-            "--threads", str(self.cfg["self_play"]["threads"])],
+            "--threads", str(self.cfg["self_play"]["threads"]),
+            "--data-entries-prefix", "i{0:04d}_".format(iter_num)],
             stderr=sys.stderr, stdout=sys.stdout, check=True)
 
     def _train(self, model_path, training_games_dir):
         logging.debug("Loading current model")
         model = self.game.load_model(model_path, self.net_type)
 
         logging.debug("Loading games by current model")
-        parser = DataParser(self.game, training_games_dir,
-                            self.cfg["training"]["entries_count"])
+        parser = DataParser(self.game, training_games_dir, self.cfg)
         train_dataset = tf.data.Dataset.from_generator(
             parser.generator, output_types=(tf.string, tf.string, tf.string))
         train_dataset = train_dataset.map(parser.get_parse_func())
         # train_dataset = train_dataset.batch(32, drop_remainder=True)
         train_dataset = train_dataset.prefetch(4)
 
         logging.info("Fitting new model...")
-        model.fit(train_dataset, epochs=4, verbose=0)
+        model.fit(train_dataset, epochs=1, verbose=0)

```

Comment:
interesting... are you sure this is better / more correct?

Reply:
not sure, but i think. As i see it, if we want to train more, we should  go over each data entry once in a training iteration, if we want to train more on the same data, we can decrease the games played in self play and make the mam loop do more iterations